src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/deathofappellant/DeathOfAppellantAboutToSubmitHandler.java

```diff
@@ -94,10 +104,27 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
         if (!shouldKeepConfidentialCaseFlag(caseDataAfter)) {
             preSubmitCallbackResponse.getData().setIsConfidentialCase(null);
         }
+        cancelHearing(callback);
 
         return preSubmitCallbackResponse;
     }
 
+    private void cancelHearing(Callback<SscsCaseData> callback) {
```

Comment:
Is there a way you could condense these down to a common method? They seem to be identical. You could have each handler define a predicate to pass into this common method.

I'd also rename the method to "sendListAssistCancellationRequest" or similar.

Reply:
As these were being worked me and Harry for different events (tickets), the duplication has taken place. As noted in comment to Nick, all handlers make use of similar conditions. There's a TODO in the code to remove the duplication after all events are implemented. I have created a Techdebt ticket SSCS-10627 in order to examine the duplicate code and remove duplication where possible, when we are past the delivery. At this point, I am inclined to live with this duplication and focus on getting tickets across the board.

Reply:
sendListAssistCancellationRequest name change method noted in the tech debt ticket now.